authlib/common/cookie_manager.py

```diff
@@ -0,0 +1,6 @@
+import streamlit as st
+import extra_streamlit_components as stx
+
+@st.experimental_singleton
+def cookie_manager():
```

Comment:
I was thinking of this being a CookieManager() class with CRUD member functions. Having it in a separate class means I (we) can change the cookie impl. If you have time to change it then please do, else just let me know and I'll approve/merge as is.

Reply:
I see what you mean. I implemented and used the CookieManager class, and Cookies have a default expiration date.

Reply:
The version raised to 1.7.0.
I went with 1.7.0 because experimental_singleton seemed to work fine in the latest version and the other functions seemed to be working as well.